server/src/main/java/org/opensearch/index/store/remote/utils/filetracker/FileType.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.store.remote.utils.filetracker;
+
+public enum FileType {
+    BLOCK,
+    NON_BLOCK;
+
+    public boolean isBlockFile(FileType fileType) {
+        return fileType.equals(FileType.BLOCK);
+    }
+}
```

Comment:
Ideally if we could cache everything at a block level that'd be great.

Reply:
As of now, we are caching everything at BLOCK level only.